--- simpletest/mock_objects.php~	2010-03-10 22:50:50.669131000 +0200
+++ simpletest/mock_objects.php	2010-03-10 22:50:57.196186467 +0200
@@ -12,11 +12,7 @@
 require_once(dirname(__FILE__) . '/expectation.php');
 require_once(dirname(__FILE__) . '/simpletest.php');
 require_once(dirname(__FILE__) . '/dumper.php');
-if (version_compare(phpversion(), '5') >= 0) {
-    require_once(dirname(__FILE__) . '/reflection_php5.php');
-} else {
-    require_once(dirname(__FILE__) . '/reflection_php4.php');
-}
+require_once(dirname(__FILE__) . '/reflection.php');
 /**#@-*/
 
 /**
--- simpletest/simpletest.php~	2010-03-10 22:51:34.046216000 +0200
+++ simpletest/simpletest.php	2010-03-10 22:51:39.102460852 +0200
@@ -9,11 +9,7 @@
 /**#@+
  * include SimpleTest files
  */
-if (version_compare(phpversion(), '5') >= 0) {
-    require_once(dirname(__FILE__) . '/reflection_php5.php');
-} else {
-    require_once(dirname(__FILE__) . '/reflection_php4.php');
-}
+require_once(dirname(__FILE__) . '/reflection.php');
 require_once(dirname(__FILE__) . '/default_reporter.php');
 require_once(dirname(__FILE__) . '/compatibility.php');
 /**#@-*/
--- simpletest/test_case.php~	2008-04-08 04:20:33.000000000 +0300
+++ simpletest/test_case.php	2010-03-10 22:52:02.519113944 +0200
@@ -17,12 +17,8 @@
 require_once(dirname(__FILE__) . '/expectation.php');
 require_once(dirname(__FILE__) . '/dumper.php');
 require_once(dirname(__FILE__) . '/simpletest.php');
-if (version_compare(phpversion(), '5') >= 0) {
-    require_once(dirname(__FILE__) . '/exceptions.php');
-    require_once(dirname(__FILE__) . '/reflection_php5.php');
-} else {
-    require_once(dirname(__FILE__) . '/reflection_php4.php');
-}
+require_once(dirname(__FILE__) . '/exceptions.php');
+require_once(dirname(__FILE__) . '/reflection.php');
 if (! defined('SIMPLE_TEST')) {
     /**
      * @ignore
--- simpletest/test/unit_tests.php~	2010-03-10 22:56:04.443267000 +0200
+++ simpletest/test/unit_tests.php	2010-03-10 22:56:12.232641760 +0200
@@ -22,15 +22,9 @@
         $this->addFile($path . '/dumper_test.php');
         $this->addFile($path . '/expectation_test.php');
         $this->addFile($path . '/unit_tester_test.php');
-        if (version_compare(phpversion(), '5', '>=')) {
-            $this->addFile($path . '/reflection_php5_test.php');
-        } else {
-            $this->addFile($path . '/reflection_php4_test.php');
-        }
+        $this->addFile($path . '/reflection_test.php');
         $this->addFile($path . '/mock_objects_test.php');
-        if (version_compare(phpversion(), '5', '>=')) {
-            $this->addFile($path . '/interfaces_test.php');
-        }
+        $this->addFile($path . '/interfaces_test.php');
         $this->addFile($path . '/collector_test.php');
         $this->addFile($path . '/adapter_test.php');
         $this->addFile($path . '/socket_test.php');
--- simpletest/test/reflection_test.php~	2010-03-10 22:58:11.002406000 +0200
+++ simpletest/test/reflection_test.php	2010-03-10 22:58:15.415874518 +0200
@@ -1,7 +1,7 @@
 <?php
 // $Id$
 require_once(dirname(__FILE__) . '/../autorun.php');
-require_once(dirname(__FILE__) . '/../reflection_php5.php');
+require_once(dirname(__FILE__) . '/../reflection.php');
 
 class AnyOldLeafClass {
 	function aMethod() { }
